channelserver/src/logging.rs

```diff
@@ -63,6 +64,7 @@ impl Actor for MozLogger {
 pub struct LogMessage {
     pub level: ErrorLevel,
     pub msg: String,
+    pub attributes: Option<HashMap<String, String>>,
```

Comment:
I always thought the logging Actor would eventually go away (couldn't a basic MozLogger somehow be passed around to places where the actor's currently used, like in session.rs?)

..I bring it up now because attributes looks potentially like a use case for slog's key value pairs instead vs interpolating them into the msg

Reply:
Yeah, revisiting logging and re-enabling metrics are on my plate for the next major PR. The actor may not go away because the SessionState object isn't always available in all the cases where logging might be useful (mostly in the meta.rs file), but that's more just a refactoring issue.